device-types/HPE/ProCurve-2910-48G.yaml

```diff
@@ -0,0 +1,112 @@
+---
+manufacturer: HPE
+model: ProCurve 2910al-48G
+slug: hpe-procurve-2910al-48g
+part_number: J9147A
+u_height: 1
+comments: '[ProCurve 2910al-48G Specs](https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c01844343)'
+is_full_depth: false
+console-ports:
+  - name: Console
+    type: rj-45
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 92
```

Comment:
Where did you get this value from? According to the datasheet, the values should be `maximum_draw: 105` and `allocated_draw: 64`

The datasheet says that the max draw is 105W and the idle power is 64W